frc53_nft/src/types.rs

```diff
@@ -7,87 +7,94 @@ use fvm_ipld_encoding::RawBytes;
 use fvm_shared::address::Address;
 use fvm_shared::ActorID;
 
+#[cfg(doc)]
+use super::state::Cursor;
```

Comment:
hah, so we have to import them for "doc" if they're not already imported here in order for the doc links to work?

Reply:
Yep: https://doc.rust-lang.org/rustdoc/advanced-features.html#cfgdoc-documenting-platform-specific-or-feature-specific-information

Well, I could also link to it with the full path, but then all the links become rather long.